packages/effect/test/Schedule.test.ts

```diff
@@ -894,11 +894,12 @@ const checkRepetitions = <Env>(schedule: Schedule.Schedule<number, number, Env>)
     return [actual, expected] as const
   })
 }
+const MAX_DATE_TIMESTAMP = 8640000000000000
 export const run = <A, E, R>(
   effect: Effect.Effect<A, E, R>
 ): Effect.Effect<A, E, R> => {
   return Effect.fork(effect).pipe(
-    Effect.tap(() => TestClock.setTime(Number.POSITIVE_INFINITY)),
+    Effect.tap(() => TestClock.setTime(MAX_DATE_TIMESTAMP)),
```

Comment:
`TestClock.setTime(Number.POSITIVE_INFINITY)` return a fail of `Invalid date`
`8640000000000000` is the max date.
Should I export `maxEpochMillis` from the internal DateTime and use it here?

Reply:
I implemented it. Feel free to revert the commit if you think it's a bad idea